scripts/dats_validator/validator.py

```diff
@@ -267,7 +267,10 @@ def validate_is_about(dataset):
     if "isAbout" in dataset.keys():
         species_present = False
         for entry in dataset["isAbout"]:
-            if "identifier" in entry.keys():
+            if (
+                "identifier" in entry.keys()
```

Comment:
@surchs does that mean that the taxonomy data present in isAbout should have a valueIRI instead of following the taxonomy schema? That piece of code here is checking that at least one species was provided in the DATS file.

Reply:
@cmadjar: no, sorry, my bad. I conflated the two tools here. I think the validator should continue checking for taxonomic information and taxonomic information requires a name and identifier. 

I think the crawler (discussed above) should not assume that all "Subject" annotations made on Zenodo are taxonomic information and rather go for the generic "annotation" schema.

The flipside is that if someone had in fact included taxonomic information among the "Subject" tags on Zenodo, then the DATS validator would still yell at them to provide this information again under `isAbout`. Not sure how to resolve this.

Reply:
@surchs I may have found a solution for that. See my latest changes. I am checking whether the identifier from Zenodo contains the NCBI taxonomy URL like in the validator. If it matches, then will encode with identifier schema, else with the annotation schema. 

Not perfect but a good start?

Reply:
Hey @cmadjar, yes, I think that's a good idea! We can add that also to our docs so we have a higher probability that folks following our guidelines will use the NCBI terms to annotate their datasets.